lib/sinatra/default_not_found_page.rb

```diff
@@ -0,0 +1,6 @@
+class Sinatra::Base
+  error Sinatra::NotFound do
+    content_type 'application/json'
+    [404, '']
```

Comment:
We should return a brief description here, otherwise it can be hard to differentiate between a "url not found" and api "not found" errors.